cem/cem.c

```diff
@@ -4036,15 +4035,19 @@ void InitNormal(void)
       Age[x][y] = 0;
     }
 
-  for (x = 0; x < initial_rock - 3; x++) {
-    for (y = 0; y < 2 * Y_MAX; y++) {
-        type_of_rock[x][y] = 'f';
-        topography[x][y] = kCliffHeightFast;
-    }
-  }
-  set_rock_blocks(type_of_rock + initial_rock - 3,
-                  topography + initial_rock - 3,
-                  X_MAX - (initial_rock - 3), 2 * Y_MAX, NUMBER_CHUNK);
+  /* TODO: fix this section
```

Comment:
There should definitely be a test somewhere up above to make sure that `initial_rock` is >= 3.